app/models/post.rb

```diff
@@ -1,3 +1,13 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :user
```

Comment:
[REQUIRED] Since the foreign key is supposed to be author_id for user model this should be `belongs_to :author, class_name: 'User'`